lib/checks/aria/aria-allowed-role.json

```diff
@@ -0,0 +1,15 @@
+{
+	"id": "aria-allowed-role",
+	"evaluate": "aria-allowed-role.js",
+	"options": {
+		"allowImplicit": true,
+		"ignoredTags": []
+	},
+	"metadata": {
+		"impact": "minor",
+		"messages": {
+			"pass": "ARIA role is allowed for given element",
+			"fail": "{{=it.data}} role{{=it.data && it.data.length > 1 ? 's are' : ' is'}} not allowed for given element"
```

Comment:
I think you'd have to do `it.data.join(', ')` so that there's a space between the roles. I would also do "Roles X, Y, Z are not allowed for given element".